npm/angular/src/mount.ts

```diff
@@ -309,6 +308,6 @@ getTestBed().initTestEnvironment(
 
 setupHooks(() => {
   // Not public, we need to call this to remove the last component from the DOM
-  TestBed['tearDownTestingModule']()
-  TestBed.resetTestingModule()
+  getTestBed()['tearDownTestingModule']()
```

Comment:
So this function isn't part of the public API for `TestBed`? Would it make sense to guard it with a `try/catch` to keep us from blowing up, or would that just break tests in a less-obvious way?

Reply:
An error thrown here makes sense, since otherwise it would lead to some really strange behavior (like not removing the previously mounted component). Since it threw, it was very easy to track down what was going on and get a fix for it.

It is marked as a [public property](https://github.com/angular/angular/pull/46635/files#diff-2bd12fb2a5773e584a5fa36c468bbdb2a76747088b0444af9ff1a3034d6c562cR678), just not exposed on the interface for some reason.